baybe/constraints/conditions.py

```diff
@@ -83,6 +105,17 @@ def evaluate(self, data: pd.Series) -> pd.Series:
             A boolean series indicating which elements satisfy the condition.
         """
 
+    @abstractmethod
+    def to_polars(self, expr: pl.Expr) -> pl.Expr:
```

Comment:
Sorry for reopening, but one last detail: would it make sense to make `expr` positional only, i.e. `to_polars(self, expr: pl.Expr, /)`?

I'm just asking because we should try to pay attention to this when writing, because it's these details that will avoid breaking changes in the future. I suspect that you'd rather want to avoid that someone passes `expr` as a keyword argument!? Because otherwise, I'd rather prefer it in non-abbreviated form. However, I think it's even better to enforce a positional-only argument here (and in the subclasses, of course)

Reply:
ok I agree and included